Frameworks/AddressBook/ABPerson.mm

```diff
@@ -200,30 +203,59 @@ bool ABPersonRemoveImageData(ABRecordRef person, CFErrorRef* error) {
 }
 
 /**
- @Status Stub
+ @Status Interoperable
  @Notes
 */
 CFIndex ABAddressBookGetPersonCount(ABAddressBookRef addressBook) {
-    UNIMPLEMENTED();
-    return StubReturn();
+    if (addressBook == nullptr) {
+        return 0;
+    }
+    _ABAddressBookManager* addressBookManager = (__bridge _ABAddressBookManager*)addressBook;
+    NSArray* contacts = [addressBookManager getListOfContacts];
+    return contacts ? [contacts count] : 0;
```

Comment:
> contacts ? [contacts count] : 0 [](start = 10, length = 32)

you can simply return [contacts count].  It is safe to pass a message to nil (and it will return 0).